cpp/ps0004/problem 5.cpp

```diff
@@ -1,26 +1,34 @@
-# include <iostream>
+#include <iostream>
 using namespace std ;
 int main ()
 {
-float number; 
+    // prompting user for number 1.5/1.5
+    float number; 
     cout << "Enter a number (must be between 32 and 126): ";
     cin >> number;
```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input